src/services/post/CreateTagService.ts

```diff
@@ -0,0 +1,26 @@
+import prismaClient from '../../prisma';
+//import { CreateTagData, ReturnTagData } from "../../interfaces/post/tagType";  <--- Coisar depois
```

Comment:
Sugestão: Retirar se não estiver sendo usado

Reply:
falta criar o tagType, vou usar essa rota quando for fazer ele